go/test/endtoend/vtgate/reservedconn/sysvar_test.go

```diff
@@ -433,30 +433,30 @@ func TestSysVarTxIsolation(t *testing.T) {
 	utils.AssertMatches(t, conn, "select @@transaction_isolation", `[[VARCHAR("REPEATABLE-READ")]]`)
 	// ensuring it goes to mysql
 	utils.AssertContains(t, conn, "select @@transaction_isolation, connection_id()", `REPEATABLE-READ`)
-	// second run, to ensuring the setting is applied on the session and not just on next query after settings.
+	// second run, ensuring it has the same value.
 	utils.AssertContains(t, conn, "select @@transaction_isolation, connection_id()", `REPEATABLE-READ`)
 
 	// setting to different value.
-	utils.Exec(t, conn, "set @@transaction_isolation = 'read-committed'")
-	utils.AssertMatches(t, conn, "select @@transaction_isolation", `[[VARCHAR("read-committed")]]`)
+	utils.Exec(t, conn, "set @@transaction_isolation = 'READ-COMMITTED'")
```

Comment:
we want to store them with upper case and return them with uppercase, but we want to allow for both cases for input, right?

Reply:
currently, we do not change the case when `set <system_variable>` is used and store it as-is. 
This is only for the case when `set transaction isolation level <value>` is provided we have the option as we convert the statement to `set @@session.transaction_isolation = <value>`

Do you suggest that I make that change as well, to store the user-provided value in upper case for `set <system_varable>`?

Reply:
Do we use the stored value for printing things, not the constants defined in sql.y?

Reply:
Changed as discussed offline.